src/consensus/replay_tower.zig

```diff
@@ -601,6 +603,14 @@ pub const ReplayTower = struct {
                 // stray slots (which should always be empty_ancestors).
                 //
                 // This is covered by test_future_tower_* in local_cluster
+                self
+                    .logger
+                    .info()
+                    .logf(
+                    "failed_switch_threshold. switch_proof_stake: {}, total_stake: {}",
+                    .{ 0, total_stake },
```

Comment:
this should at least be warn since it represents a bad state. if it requires action, then it should be error